fastlane/Fastfile

```diff
@@ -43,10 +43,14 @@ platform :android do
 		rollout='0.2'
 	end
 
+    versions = google_play_track_version_codes(track: "internal")
+    version_code = versions[0]
+
 	supply(
 		track: 'internal',
 		track_promote_to: "production",
-		rollout: rollout
+		rollout: rollout,
+		version_code: version_code
```

Comment:
Uploading metadata between `publish` and `promote` requires specifying the `version_code`. We could parse it from the `.gradle` file, but this is easier.

Some tabs/spaces inconsistencies in this file which we can ignore for now.